shared/chat/conversation/messages/set-explode-popup/index.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import * as React from 'react'
+import {messageExplodeDescriptions} from '../../../../constants/chat2'
```

Comment:
lets try and make these components very dumb. aka the descriptions should come in simply as an own prop (when integrated in the connector), and it'd be sorted, etc